templates/p5_gfd.bt

```diff
@@ -394,11 +402,7 @@ typedef struct(u32 version)
     u32 Field6C;
     u32 Field70;
     s16 Field50;
-    
-    if ( version <= 0x1105070 || version >= 0x1105090 )
-    {
-        u32 Field98;
-    }
+	u32 Field98;
```

Comment:
why is the version check removed

Reply:
This version check is redundant, if you look at the comparison, this will basically always be true, so might as well remove the redundancy.

Reply:
It's not redundant as it's there for preservations sake.